packages/netlify-cms-backend-github/src/API.js

```diff
@@ -115,18 +119,12 @@ export default class API {
     return `${this.repo}/${collectionName}/${slug}`;
   }
 
-  generateBranchNameFromCollectionAndSlug(collectionName, slug) {
```

Comment:
Unused method, and in any case `generateContentKey` doesn't return a promise so this method will fail.

Reply:
Removed in #2523: https://github.com/netlify/netlify-cms/pull/2523/commits/3daa87dc954e95e3a784b52b64bac26f0a8c8f6b